ui/DesignSystem/Primitive/Icon/CheckScallop.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
```

Comment:
I changed my mind about the file name. I think we should call it "Badge.svelte".

## Badge:
<img width="1021" alt="Screenshot 2020-02-26 at 09 59 23" src="https://user-images.githubusercontent.com/158411/75328721-bd1adf80-587e-11ea-9aca-321edcf07316.png">

## vs scallop:

<img width="975" alt="Screenshot 2020-02-26 at 09 59 38" src="https://user-images.githubusercontent.com/158411/75328739-c146fd00-587e-11ea-8cda-4064ac4d5131.png">

Reply:
😃 you convinced me!